### PR TITLE
To Do List: List structure 

### PR DESCRIPTION
**Pull request by degisew**
### To-Do list: list structure
**In this project, I implemented the following features:**
- I created a new project with webpack.
- I created an index.html file and wrote my HTML markup there. 
- I settled the index.html file as a template using the HTML Webpack Plugin.
- I created an index.js file and set an array of some simple to-do tasks and each task object contains three keys: description 
[string], completed [bool], and index: [number].
- I created a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load, the dynamically created list of tasks in the dedicated placeholder is rendered. The list appears in order of the index values for each task.
I created a style.css and I set rules for the To-Do List. 
- CSS  is loaded by Webpack Style/CSS Loader.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+  /* font-size: 15px; */
+  font-weight: normal;
+  column-gap: 10px;
+}
+
+.tasks {
+  display: block;
+
+  /* background-color: green; */
```
Comment (Davidosky007) [inline src/style.css]: * Kindly remove all commented code from your codebase as it makes your code look untidy and unprofessional 🙏

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+ul li {
+  display: flex;
+  width: 100%;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid rgb(214, 212, 212);
+
+  /* font-size: 15px; */
```
Comment (Davidosky007) [inline src/style.css]: * Kindly remove all commented code from your codebase as it makes your code look untidy and unprofessional 🙏

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+.each-task-btn {
+  border: none;
+  width: 20px;
+
+  /* outline: none; */
+
+  /* font-size: 20px; */
+
+  /* font-weight: bolder; */
```
Comment (Davidosky007) [inline src/style.css]: * Kindly remove all commented code from your codebase as it makes your code look untidy and unprofessional 🙏

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+  background-color: inherit;
+  background-image: url(./assets/dotsLoading.png);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: contain;
+
+  /* width:fit-content; */
+
+  /* height: fit-content; */
```
Comment (Davidosky007) [inline src/style.css]: * Kindly remove all commented code from your codebase as it makes your code look untidy and unprofessional 🙏

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+#clear {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+
+  /* margin-top: 20px; */
```
Comment (Davidosky007) [inline src/style.css]: * Kindly remove all commented code from your codebase as it makes your code look untidy and unprofessional 🙏

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import './style.css';
+import Img from './assets/rotateArrow.png';
+import enter from './assets/enter.png';
+
+const tasksContainer = document.getElementById('tasks');
+const rotateArrow = document.getElementById('rotate-arrow');
+const enterIcon = document.getElementById('enter-icon');
+// const titleDiv = document.querySelector('.title');
```
Comment (Davidosky007) [inline src/index.js]: * Kindly remove all commented code from your codebase as it makes your code look bulky and unprofessional 🙏

File: src/index.js [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    index: 1,
+  },
+  {
+    description: 'washing clothes',
+    completed: false,
+    index: 2,
+  },
+];
+
+//  <div class="each-list-container"></div>
```
Comment (Davidosky007) [inline src/index.js]: * Kindly remove all commented code from your codebase as it makes your code look bulky and unprofessional 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @degisew 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good  looking readme file 👍 
* Proper use of Github flow 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 7 above)
Comment (degisew) [reply src/index.js]: Ok, thank you. I will walk through it.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**